pyblish_ftrack/plugins/conform_ftrack_component_preview.py

```diff
@@ -0,0 +1,43 @@
+import pyblish.api
+import os
+import ftrack
+
+@pyblish.api.log
+class FtrackUploadPreview(pyblish.api.Conformer):
+    """Creates Preview component within supplied version, also makes this preview reviewable.
+
+    Expected data members:
+    'publishedFile' - path that will be saved as a component
+    'ftrackVersionID' - ID of a version where component should be created
```

Comment:
This is really good.

I've been considering whether to introduce a standard format in which to document these types of members, similar to how Sphinx assumes a certain format such that it can render it nicely. We could then use that in the GUI and upcoming plug-in documentation generator/frontend.